# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/gtksharp/hotfix-inst.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- GtkSharp-3.22.2/Source/gio/generated/meson.build.vanilla    2026-09-12 20:55:37.529960117 +0300
+++ GtkSharp-3.22.2/Source/gio/generated/meson.build    2026-09-12 20:57:28.227569099 +0300
@@ -413,5 +413,5 @@
 source_gen = files(generated_sources)
 gio_api_includes = join_paths(meson.current_source_dir(), 'gio-api.xml')
 if install
-    install_data(gdk_api_includes, install_dir: gapi_xml_installdir)
+    install_data(gio_api_includes, install_dir: gapi_xml_installdir)
 endif

